src/exceptions.js

```diff
@@ -12,5 +12,9 @@ export class ExtensibleError extends Error {
 }
 
 export class DuplicateZipEntryError extends ExtensibleError {}
-export class RDFParseError extends ExtensibleError {}
+
```

Comment:
I'd got rid of this space as it seemed unecessary - but can add that back if you prefer.